requirements/edx-sandbox/py35.in

```diff
@@ -15,15 +15,15 @@
 -c ../constraints.txt
 
 -r shared.txt                       # Dependencies in common with LMS and Studio
-chem
+chem                                # A helper library for chemistry calculations
 matplotlib==2.2.4                   # 2D plotting library
 networkx==2.2                       # Utilities for creating, manipulating, and studying network graphs
 numpy==1.16.5                       # Numeric array processing utilities; used by scipy
-openedx-calc
+openedx-calc<2.0.0
```

Comment:
Note: Added this out of compatibility concerns though if this is desired, we should probably do the same with `chem`